security-openid-connect-web-authentication-quickstart/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <properties>
         <quarkus.version>999-SNAPSHOT</quarkus.version>
-        <keycloak.version>7.0.1</keycloak.version>
+        <keycloak.version>9.0.0</keycloak.version>
```

Comment:
Is this property used, I don't see it used?

Reply:
@gsmet sorry, indeed, it is in the code, let me update a bit

Reply:
@gsmet Update the code in two quaickstarts to check the system property